dbh.inc.php

```diff
@@ -0,0 +1 @@
+
```

Comment:
@yly828 Is this file supposed to be empty? If it's not in use, please delete it.